lib/carrierwave/processing/rmagick.rb

```diff
@@ -351,5 +373,9 @@ def destroy_image(image)
       image.destroy! if image.respond_to?(:destroy!)
     end
 
+    def rmagick_image
+      ::Magick::Image.ping(current_path).first
```

Comment:
I'm a little concerned about this. Calling `ping` will return an image with invalid pixel data, and naming this `rmagick_image` seems to be asking for trouble.

Either we need to read the entire image, or we should rename the method.